libraries/src/MVC/Model/BaseDatabaseModel.php

```diff
@@ -177,12 +177,12 @@ protected function _getListCount($query)
 		}
 
 		// Otherwise fall back to inefficient way of counting all results.
-
-		// Remove the limit and offset part if it's a DatabaseQuery object
+		
+		// Remove the limit, offset and order parts if it's a DatabaseQuery object
```

Comment:
```suggestion

		// Remove the limit, offset and order parts if it's a DatabaseQuery object
```
Please remove the tabs in the empty line 180 (but not the line itself) to satisfy our code style checker.